tests/Unit/database/migrations/CreateUsersTableTest.php

```
@@ -104,7 +104,11 @@ public function it_checks_if_users_table_column_types(): void
         }
 
         // Test boolean column - PostgreSQL returns 'bool'
-        $this->assertContains(Schema::getColumnType('users', 'is_active'), ['boolean', 'bool']);
+        if (config('database.default') === 'pgsql') {
+            $this->assertEquals('bool', Schema::getColumnType('users', 'is_active'));
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Schema' in method 'it_checks_if_users_table_column_types'. (Clean Code Rules)

(StaticAccess)